js/dataset/display.js

```diff
@@ -48,36 +48,38 @@ function prepare_resources() {
         });
 
         // Check asynchronuously the status of displayed resources
-        $this.find('.format-label').each(function() {
```

Comment:
Did you change anything except the indent? Not sure if that's github messing up with the diff or if there are actual changes.